scripts/android.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
This file needs updates regarding SDK version, shared aar and getting aars from Maven.